Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx

```diff
@@ -188,6 +199,7 @@ void ITSMFTDeadMapBuilder::run(ProcessingContext& pc)
   mFirstOrbitTF = pc.services().get<o2::framework::TimingInfo>().firstTForbit;
 
   if (mFirstOrbitRun == 0x0) {
+    mRunNumber = pc.services().get<o2::framework::TimingInfo>().runNumber;
     mFirstOrbitRun = mFirstOrbitTF;
   }
```

Comment:
I would suggest to move the `isEnded` check below and here to check:
```
const auto& tinfo = pc.services().get<o2::framework::TimingInfo>();
if (tinfo.globalRunNumberChanged || mFirstOrbitRun == 0x0) { // new run is starting
  mRunNumber = tinfo.runNumber;
  mFirstOrbitRun = tinfo.firstTForbit;
  isEnded = false;
}
if (isEnded) {
  return;
}
mFirstOrbitTF = tinfo.firstTForbit;
```
In this way the `run()` will be ready to start/stop/start